include/kinematics/FrameDisplay.m

```diff
@@ -55,16 +55,18 @@
     %         end
     %     end
     
-    properties (Access = private)
+    properties (Access = public)
```

Comment:
Is there a reason for making everything public rather than adding a new property section? The internal axHandle, figHandle etc. should still remain private, right?
 
```
properties (Access = public)
    drawLinks = true;
end
```

Reply:
I was also fiddling with changing axis limits and frame axes lengths on the fly.  I just made things public instead of doing getters and setters.

Reply:
I wasn't advocating for getters/setters. In MATLAB classes you can have multiple `properties` sections, e.g., a private and a public one.